mode-lambda/backend/gl/combine.fragment.glsl

```diff
@@ -59,7 +59,7 @@ void main() {
         && (0.0 <= ex && ex <= width)) {
       fin_Color.rgb =
           fin_Color.rgb * (1.0 - lay_Color.a)
-        + lay_Color.rgb * lay_Color.a;
+        + lay_Color.rgb;
```

Comment:
If I am reading the last equation here --- https://en.wikipedia.org/wiki/Alpha_compositing#Description --- correctly then we can do `fin_Color = lay_Color + fin_Color * (1 - lay_Color.a)` and make sure that the content of the layer is also premultiplied-alpha

Reply:
I think you are right - I'll change it and test.